src/frontend/src/components/SearchInput/SearchInput.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles((theme) => ({
+  input: {
+    fontSize: '1.6rem',
+    '&:hover': {
+      border: '2px solid',
+      borderColor: theme.palette.background.default,
+    },
+    '&:focus': {
+      border: '2px solid',
+      borderColor: theme.palette.primary.main,
+    },
+    '& > *': {
+      fontSize: '1.6rem !important',
+      color: theme.palette.text.default,
+    },
+    height: '55px',
+    backgroundColor: theme.palette.background.default,
+    paddingLeft: '10px',
+    // The border around the textField inputs was a default of the type so I had to add my own
```

Comment:
This looks duplicated from the other file, do we need this definition in both places?  Seems like we only need to apply it once (i.e, the whole `input` definition).

Reply:
Yep just tested it, there's no need for this code. Removed it.